packages/server/src/routes/grants.js

```diff
@@ -491,6 +459,38 @@ router.get('/:grantId/notes', requireUser, async (req, res) => {
     res.json(rows);
 });
 
+// getting notes for a specific user and grant
+router.get('/:grantId/notes/user/:userId', requireUser, async (req, res) => {
+    const { grantId, userId } = req.params;
+    const { tenant_id: organizationId } = req.session.user;
+    const { cursor, limit } = req.query;
+
+    // Converting limit to an integer
+    const limitInt = limit ? parseInt(limit, 10) : undefined;
+
+    // Validating the limit query parameter
+    if (limit && (!Number.isInteger(limitInt) || limitInt < 1 || limitInt > 100)) {
+        res.status(400).send('Invalid limit parameter');
+        return;
+    }
+
+    try {
+        // Fetching the notes using getOrganizationNotesForGrantByUser function
+        const notes = await getOrganizationNotesForGrantByUser(
+            knex,
+            organizationId,
+            userId,
+            grantId,
+            { cursor, limit: limitInt },
+        );
+
+        // sending the notes as JSON response
+        res.json(notes);
+    } catch (error) {
+        res.status(500).json({ error: 'Failed to retrieve notes' });
```

Comment:
Is it possible to re-throw `error` on the next line so that we can preserve the error/trace details, or will that mess with the ability to provide the `Failed to retrieve notes` message in the response?

Reply:
yes, I removed the try..catch here as we aren't using this message on the FE and will still result in a 500